src/Language/Haskell/Ghcid/Util.hs

```diff
@@ -85,8 +85,16 @@ ignored act = do
     waitBarrier bar
 
 -- | The message to show when no errors have been reported
-allGoodMessage :: String
-allGoodMessage = setSGRCode [SetColor Foreground Dull Green] ++  "All good" ++ setSGRCode []
+allGoodMessage :: [String] -> String
+allGoodMessage target = setSGRCode [SetColor Foreground Dull Green] ++ "All good " ++ setSGRCode [] ++ "in " ++ concat target 
```

Comment:
If target is blank, as it often will be, this will look really odd - i suspect you need to hide the "in " in that case.